src/webextension/background/views.js

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const views = new Map();
+export async function openView(name) {
```

Comment:
In the interests of simplicity (and since we haven't totally decided whether to force having only one tab per tab-type), I think we could skip doing this for now. If UX definitely wants it this way though, I guess I'm ok with it.

Reply:
The latest changes treat "manage" and "firstrun" differently.